### ISSUE TITLE
spider refinament does not create any output

### ISSUE THREAD

**Issue by rmarabini** [I2PC/scipion#56]

**Comment by rmarabini** [comment]
No symmetry is asked in spider refinement!

**Comment by azazellochg** [comment]
the protocol crashes at the very beginning, when running [prepare.pam](scipion/pyworkflow/em/packages/spider/scripts/projmatch/Refinement/prepare.pam) script:

```
  -- START OF: prepare.pam    --
 .OPERATION: [iter]=[iter1]  ; Needed for naming initial files
 .OPERATION: VM  
 .SYSTEM COMMAND: 
     mkdir -p work/ final/ input/
 .OPERATION: CP  ; Copy volume
 .INPUT FILE NAME OR TEMPLATE (E.G. STK@****):  ../Reconstruction/vol01 
 *** FILE NOT FOUND: ../Reconstruction/vol01.stk
 *** ERROR: OPENFIL; FILE NOT FOUND: ../Reconstruction/vol01.stk 
```

**Comment by azazellochg** [comment]
Also, spider refinement protocol fails if ctf estimation was not done, since it needs defocus groups

**Comment by delarosatrevin** [comment]
@azazellochg this is an easy one, we could put a condition in the inputParticles to allow only the ones which haveCTF.

**Comment by azazellochg** [comment]
fixed in cd443bc
